R/jitter.R

```diff
@@ -244,15 +243,14 @@ jitter <- function(dir = getwd(),
 #' @return Negative log-likelihood of one jittered model
 #'
 iterate_jitter <- function(i,
-                           dir = getwd(),
                            printlikes = TRUE,
                            exe = "ss",
                            verbose = FALSE,
                            init_values_src = 0,
                            ...) {
-  jitter_dir <- file.path(dir, paste0("jitter", i))
+  jitter_dir <- paste0("jitter", i)
   copy_SS_inputs(
-    dir.old = dir, dir.new = jitter_dir, overwrite = TRUE,
+    dir.old = getwd(), dir.new = jitter_dir, overwrite = TRUE,
```

Comment:
@okenk maybe dir should be a param, possible `dir = dir` where it uses the dir from the wrapper function and then `dir.old = dir` in case someone is trying to use this function for a model not in their current working directory?

Reply:
You cannot pass it the directory from the wrapper function because that is already the working directory, set early in `jitter()`, so it would essentially look for base_jitter_dir/base_jitter_dir, which doesn't exist. That is what caused the error you and Ian were getting. That error doesn't pop up if you call `jitter()` from base_jitter_dir instead of passing it a directory argument, because then the directory is essentially empty.

This is a very non-trivial problem, but after quite a bit of trial and error, I figured out how pass it a *relative* directory, which I haven't pushed yet. Allowing the user to pass an absolute directory would require even more work that I am not up for. But I'm sure someone else could figure it out.

Let me know how you'd like to proceed.

Reply:
Okay, that makes sense. I'm good with your changes but I'll let Ian review it as well and give his approval. Thanks for your work on this!